apps/iiif/manifests/views.py

```diff
@@ -63,6 +63,38 @@ def get(self, request, *args, **kwargs): # pylint: disable = unused-argument
                 safe=False
             )
 
+class AllVolumesCollection(View):
+    """Endpoint for all volumes manifest."""
```

Comment:
For consistency, maybe "all volumes collection" in this docstring

Reply:
Updated as suggested